addons/web/static/src/views/kanban/kanban_renderer.js

```diff
@@ -56,6 +55,8 @@ export class KanbanRenderer extends Component {
         "noContentHelp?",
         "scrollTop?",
         "canQuickCreate?",
+        "quickCreateState?",
```

Comment:
I think this is mandatory

```suggestion
        "quickCreateState",
```